onnxruntime/core/session/provider_bridge_ort.cc

```diff
@@ -1800,8 +1801,21 @@ std::shared_ptr<IExecutionProviderFactory> OpenVINOProviderFactoryCreator::Creat
   return s_library_openvino.Get().CreateExecutionProviderFactory(&ov_options_converted_map);
 }
 
-std::shared_ptr<IExecutionProviderFactory> OpenVINOProviderFactoryCreator::Create(const ProviderOptions* provider_options_map) {
-  return s_library_openvino.Get().CreateExecutionProviderFactory(provider_options_map);
+ProviderOptions ORTSessionOptionsToOrtOpenVINOProviderOptions(const ProviderOptions ov_options, const SessionOptions *session_options) {
```

Comment:
@preetha-intel  I'd recommend to use below, its better to pass a reference or a pointer to avoid any copies -

`ProviderOptions ORTSessionOptionsToOrtOpenVINOProviderOptions(const ProviderOptions& ov_options, const SessionOptions *session_options)`